Source/WebCore/platform/graphics/gstreamer/VideoFrameGStreamer.h

```diff
@@ -55,7 +55,7 @@ class VideoFrameGStreamer final : public VideoFrame {
 
     RefPtr<VideoFrameGStreamer> resizeTo(const IntSize&);
 
-    GstSample* sample() const { return m_sample.get(); }
+    GstSample* sample() const final { return m_sample.get(); }
```

Comment:
Is it necessary to virtualize this method? In VideoFrame::paintInContext() you could assert `isGStreamer()` and downcast down to `VideoFrameGStreamer`.  Type-casting macros would be useful, but they're not there (yet).